chart/templates/post-delete-hook-configMap.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.postDelete.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.postDelete.name}}-script
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "rancher.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": post-delete
+    "helm.sh/hook-weight": "1"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded,hook-failed
+data:
+  post-delete-hook.sh: |-
+{{ $.Files.Get "scripts/post-delete-hook.sh" | indent 4 }}
+{{ end }}
```

Comment:
```suggestion
{{- end }}
```

Reply:
same comment for wherever else the prefix `-` is omitted.

While it's not a big deal, if someone were to add a resource under this it could potentially leave weird newlines in the helm template produced.

Reply:
fixed all occurrences.